GeoQuizChallenge/app/src/main/java/com/bignerdranch/android/geoquiz/Views/QuizActivity.java

```diff
@@ -0,0 +1,154 @@
+package com.bignerdranch.android.geoquiz.Views;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+import android.util.Log;
+import android.view.Gravity;
+import android.widget.Button;
+import android.widget.ImageButton;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.bignerdranch.android.geoquiz.Controllers.QuestionController;
+import com.bignerdranch.android.geoquiz.Models.Question;
+import com.bignerdranch.android.geoquiz.R;
+
+import java.util.HashSet;
+import java.util.List;
+
+import androidx.annotation.StringRes;
+
+public class QuizActivity extends AppCompatActivity {
+  private static final String TAG = "QuizActivity";
+  private static final String KEY_INDEX = "index";
+
+  @SuppressWarnings("FieldCanBeLocal")
+  private Button trueButton;
+  @SuppressWarnings("FieldCanBeLocal")
+  private Button falseButton;
+  private ImageButton nextButton;
+  private ImageButton prevButton;
+  private TextView questionTextView;
+  private List<Question> questionBank = QuestionController.getInstance().getQuestionBank();
+  private int currentIndex = 0;
+  private int countOfCorrectQuestionsResolved = 0;
+  private int countOfQuestionsResolved = 0;
+  private HashSet<Question> resolvedQuestions = new HashSet();
```

Comment:
I think you can define it as set
```suggestion
  private Set<Question> resolvedQuestions = new HashSet();
```